tests/utils/test_databricks_utils.py

```diff
@@ -49,7 +49,7 @@ def test_databricks_params_custom_profile(ProfileConfigProvider):
     mock_provider.get_config.return_value = \
         DatabricksConfig("host", "user", "pass", None, insecure=True)
     ProfileConfigProvider.return_value = mock_provider
-    params = databricks_utils.get_databricks_host_creds("profile")
+    params = databricks_utils.get_databricks_host_creds("databricks://profile")
```

Comment:
did you want to use `construct_db_uri_from_profile` here too?

Reply:
Ah yes! Good point